journal-of-international-relations-and-development.csl

```diff
@@ -0,0 +1,223 @@
+      </group>
+    </layout>
+  </citation>
+  <bibliography hanging-indent="true" et-al-min="6" et-al-use-first="1">
+    <sort>
+      <key macro="author"/>
+      <key variable="year-suffix"/>
+      <key variable="issued"/>
+    </sort>
+    <layout suffix=".">
+      <group delimiter=" ">
+        <text macro="author"/>
+      </group>
+      <choose>
+        <if type="article-newspaper article-magazine" match="any">
+          <group delimiter=" ">
+            <text macro="title" prefix=" " suffix="."/>
+          </group>
+          <group delimiter=", " prefix=" ">
+            <text variable="container-title" font-style="italic"/>
+            <text macro="day-month"/>
+            <text variable="edition"/>
+            <text variable="page"/>
+          </group>
+        </if>
+        <else-if type="report" match="any">
+          <text macro="year-date" prefix=" (" suffix="). "/>
+          <text macro="title"/>
+        </else-if>
+        <else-if type="thesis">
+          <text macro="title" prefix=" " suffix="."/>
+          <group prefix=" " delimiter=", ">
+            <text macro="edition"/>
+            <text macro="editor" suffix="."/>
+            <text variable="genre"/>
+            <text macro="publisher"/>
+          </group>
+        </else-if>
+        <else-if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+          <group delimiter=" ">
+            <text macro="year-date" prefix=" (" suffix=")"/>
+            <text macro="title" prefix=" " suffix=","/>
+            <text macro="edition"/>
+            <text macro="editor"/>
+            <text macro="publisher"/>
+          </group>
+        </else-if>
+        <else-if type="chapter paper-conference" match="any">
+          <group delimiter=" ">
+            <text macro="year-date" prefix=" (" suffix=")"/>
+            <text macro="title" prefix=" " suffix=","/>
+            <group delimiter=", " suffix=",">
+              <group delimiter=" " prefix=" ">
+                <text term="in"/>
+                <text macro="editor" text-case="lowercase"/>
+                <text variable="container-title" font-style="italic"/>
+              </group>
+              <group>
+                <text variable="page"/>
+              </group>
+              <text variable="volume"/>
+              <text variable="collection-title" font-style="italic"/>
+            </group>
+            <text macro="publisher"/>
+          </group>
+        </else-if>
+        <else>
+          <group suffix=",">
+            <text macro="year-date" prefix=" (" suffix=")"/>
+            <text macro="title" prefix=" "/>
+            <text macro="editor" prefix=" "/>
+          </group>
+          <group prefix=" ">
+            <text variable="container-title" font-style="italic"/>
+            <group>
+              <text variable="volume" prefix=" "/>
+              <text variable="issue" prefix="(" suffix="):"/>
```

Comment:
I agree but I was not able to figure out how to do it... could you help me here? It does make sense what you are saying.